package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "meanjs",
   "description": "Full-Stack JavaScript with MongoDB, Express, AngularJS, and Node.js.",
-  "version": "0.4.2",
-  "meanjs-version": "0.4.2",
+  "version": "0.5.0-beta",
```

Comment:
It doesn't really matter too much to me, the whole point is to just tag it as different so people know.